src/drivers/abstract/attribute.js

```diff
@@ -106,7 +106,7 @@ class Attribute {
 
   equals(targetAttribute) {
     if (!targetAttribute) return false;
-    const props = [ 'dataType', 'allowNull', 'defaultValue', 'primaryKey' ];
```

Comment:
`attribute.defaultValue` were left unset on purpose while migrating skylark models, because currently leoric will try to set attribute with the defaultValue when saving records. Theoretically, there is no need to set these values but since there might be existing code that relies on this behavior, let's keep it this way for now.

There might be models that have attribute.defaultValue set with values different than the actual default in database. When we're sure that there's none, we can add this check back.

Reply:
Agreed, we should avoid inserting values by default.